config/initializers/aws.rb

```diff
@@ -0,0 +1,5 @@
+Aws.config.update(
+  region: Figaro.env.aws_region,
```

Comment:
I would've preferred to see `.aws_region!` so this will be required to be set.

Reply:
Good point. I was just copying from the IdP, but that doesn't mean I can't do better. I'll get that in another round.